modules/core/src/test/scala/org/scalasteward/core/edit/EditAlgTest.scala

```diff
@@ -27,21 +27,26 @@ class EditAlgTest extends FunSuite {
     val update = ("org.typelevel".g % "cats-core".a % "1.2.0" %> "1.3.0").single
     val file1 = repoDir / "build.sbt"
     val file2 = repoDir / "project/Dependencies.scala"
+    val gitignore = repoDir / ".gitignore"
 
     val state = MockState.empty
-      .addFiles(file1 -> """val catsVersion = "1.2.0"""", file2 -> "")
+      .addFiles(file1 -> """val catsVersion = "1.2.0"""", file2 -> "", gitignore -> "")
```

Comment:
**suggestion**: It would be nice if `gitignore` here wouldn't be empty and there would be a file that would be read if `gitignore` would be empty to test that we actually ignore files ignored by git.

Reply:
I added a `target/` entry to the gitignore file then added this file `target / Somefile.scala`  and the test confirms it is not read.

Reply:
Thanks!